src/components/header.component.vue

```diff
@@ -0,0 +1,395 @@
+          >
+            <img src="../assets/search_icon.svg" alt="/" />
+          </button>
+        </form>
+        <button
+          class="Search__CancelButton"
+          v-if="isShowCategory"
+          v-on:click="
+            isShowCategory = false
+            isShowCartText = true
+          "
+        >
+          취소
+        </button>
+      </div>
+      <div class="CartContainer" v-if="isShowCartText">
+        <div class="CartLink__Wrapper">
+          <a href="/cart" class="CartLink">장바구니</a>
+        </div>
+      </div>
+    </div>
+    <div class="MainSearchContainer__Wrapper" v-show="isShowCategory">
+      <div class="MainSearchContainer">
+        <div class="SearchHistory__Container">
+          <div class="SearchHistory__Header">
+            <h3 class="SearchHistory__Text">추천 검색어</h3>
+          </div>
+          <div class="RecommendSearchKeywordList__Container">
+            <button
+              class="CandidateItem__Button"
+              v-for="recommended in DummyData.recommended"
+              :key="recommended"
+            >
+              <div class="CandidateText">{{ recommended }}</div>
+            </button>
+          </div>
+          <div class="Divider__Container">
+            <hr color="#f8f8f8" class="Divider__Line" />
+          </div>
+          <div class="SearchHistory__HeaderRow">
+            <h3 class="SearchHistory__Text">
+              인기 검색어
+              <div class="SearchHistory__GeneratedAt">
+                <img src="../assets/Time_icon.svg" />
+                <div class="SearchHistory__GeneratedAtText">13:15 기준</div>
+              </div>
+            </h3>
+          </div>
+          <div class="PopularSearchKeywordList__Row">
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(-5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number" v-if="index < 5">{{
+                  index + 1
+                }}</span>
+                <span class="CandidateItem__Text" v-if="index < 5">{{
+                  Item
+                }}</span>
+              </div>
+            </div>
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number">{{ index + 6 }}</span>
+                <span class="CandidateItem__Text">{{ Item }}</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Header',
+  data: () => {
+    return {
+      isShowCategory: false,
+      isShowCartText: true,
+      DummyData: {
```

Comment:
보통 상수는 전부 다 대문자로 쓰고 언더바(_)로 띄어쓰기 구분을 해서 DUMMY_DATA라고 하기도 합니다.
그런데 이 경우에는 일반 변수처럼 카멜 케이스로 작성해주시는게 좋을 것 같아요

```suggestion
      dummyData: {
```